src/utils/data_types.h

```diff
@@ -14,7 +14,6 @@
 #include <cassert>
 
 #include "global_params.h"
-#include "openmvg_descriptor.hpp"
```

Comment:
Please wait my next PR that will come in some hours.
My next PR will require some coding style fix from you. Will try to do best for the other ones.